### ISSUE TITLE
Call button

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#13]
## User story
1. As a that creates a ticket in the WebApp
2. I want / need a button that automatically starts a call to the tool.
3. So that I can enter the information via speech

## Acceptance criteria
* After clicking the button the call starts immediately
* The button is easy to spot

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
